core/domain/classifier_domain_test.py

```diff
@@ -108,30 +110,171 @@ def test_validation(self):
         training_job = self._get_training_job_from_dict(training_job_dict)
         training_job.validate()
 
-        # Verify validation error is raised when int is provided for instance id
+        # Verify validation error is raised when int is provided for exp_id
```

Comment:
Each of these comment "headers" can be split up into their own test case. Please do so.

Reply:
Done